docs/source/getting_started.rst

```diff
@@ -0,0 +1,159 @@
+Getting Started
+===============
+
+The following section will cover basic concepts and usage patterns in ``unstructured``.
+After reading this section, you should be able to:
+
+* Partitiong a document with the ``partition`` function.
+* Understand how documents are structured in ``unstructured``.
+* Convert a document to a dictionary and/or save it as a JSON.
+
+The example documents in this section come from the
+`example-docs <https://github.com/Unstructured-IO/unstructured/tree/main/example-docs>`_
+directory in the ``unstructured`` repo.
+
+Before running the code in this make sure you've installed the ``unstructured`` library
+and all dependencies using the instructions in the quick start section.
+
+
+#######################
+Partitioning a document
+#######################
+
+In this section, we'll cut right to the chase and get to the most important part of the library: partitioning a document.
+The goal of document partitioning is to read in a source document, split the document into sections, categorize those sections,
+and extract the text associated with those sections. Depending on the document type, unstructured uses different methods for
+partitioning a document. We'll cover those in a later training notebook. For now, we'll use the simplest API in the library,
+the ``partition`` function. The ``partition`` function will detect the filetype of the source document and route it to the appropriate
+partitioning function. You can try out the partition function by running the cell below.
+
+
+.. code:: python
+
+
+	from unstructured.partition.auto import partition
+
+	elements = partition(filename="example-10k.html")
+
+
+You can also pass in a file as a file-like object using the following workflow:
+
+
+.. code:: python
+
+	with open("example-10k.html", "rb") as f:
+	    elements = partition(file=f)
+
+
+The ``partition`` function uses `libmagic <https://formulae.brew.sh/formula/libmagic>`_ for filetype detection. If ``libmagic`` is
+not present and the user passes a filename, ``partition`` falls back to detecting the filetype using the file extension.
+``libmagic`` is required if you'd lke to pass a file-like object to ``partition``.
```

Comment:
Add a disclaimer that very different behaviors may be observed with and without libmagic?